integrations/editor_test.go

```diff
@@ -104,3 +106,55 @@ func TestCreateFileOnProtectedBranch(t *testing.T) {
 	// Check body for error message
 	assert.Contains(t, string(resp.Body), "Can not commit to protected branch &#39;master&#39;.")
 }
+
+func testEditFile(t *testing.T, session *TestSession, urlstr string) {
```

Comment:
I would rather change this to `owner, repo, branch, filePath, content string` TBH